### ISSUE TITLE
Don't see the "Create Epic" button on the New issue screen in zenhub

### ISSUE THREAD

**Issue by nshahani** [ZenHubIO/support#1358]
I used to be able to see the "Create Epic"/"Convert to Epic" button on the New Issue screen... but, I don't see it anymore! Uninstalled the Zenhub plugin and reinstalled it, but that did not help. So I am unable to create any Epics. 

Any idea how to get that button to appear again?

**Comment by kendramoroz** [comment]
Hey @nshahani - sounds like you don't have the extension installed! If you need to grab it, you can head to https://zenhub.com/extension if you're using GitHub.com, or if you're using GitHub Enterprise with ZenHub Enterprise, you can reach out via support@zenhub.com for the link unique to your team :) 

Let me know!

**Comment by kendramoroz** [comment]
Hey @nshahani are you still having any trouble getting Epics appearing :)?

**Comment by nshahani** [comment]
Hello Kendra

No issues anymore. I had the right plugin installed, but was not logged 
into Zenhub enterprise. Once I logged in to Zenhub enterprise I started 
seeing the Epic option, board etc. 

Thanks for following up. 



Regards,

Neha Shahani
Application Architect & Technical Lead
Global Solution Services - Custom Digital Solutions
Global Technology & Integration - IBM Digital Business Group


Mobile: 1-704-763-9778 | Phone: 1-704-644-0838
E-mail: shahani@us.ibm.com
GSS team page: ibm.biz/gssportfolio
